src/Symfony/Component/Mailer/Bridge/Azure/Transport/AzureApiTransport.php

```diff
@@ -0,0 +1,336 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Mailer\Bridge\Azure\Transport;
+
+use Psr\EventDispatcher\EventDispatcherInterface;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Mailer\Envelope;
+use Symfony\Component\Mailer\Exception\HttpTransportException;
+use Symfony\Component\Mailer\SentMessage;
+use Symfony\Component\Mailer\Transport\AbstractApiTransport;
+use Symfony\Component\Mime\Address;
+use Symfony\Component\Mime\Email;
+use Symfony\Contracts\HttpClient\Exception\DecodingExceptionInterface;
+use Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+use Symfony\Contracts\HttpClient\ResponseInterface;
+
+class AzureApiTransport extends AbstractApiTransport
+{
+    private const HOST = '%resource_name_dot%communication.azure.com';
+
+    /**
+     * User Access Key from Azure Communication Service (Primary or Secondary key).
+     */
+    private string $key;
+
+    /**
+     * The endpoint API URL to which to POST emails to Azure
+     * https://{acsResourceName}.communication.azure.com/.
+     */
+    private string $resourceName;
+
+    /**
+     * The version of API to invoke.
+     */
+    private string $apiVersion = '2023-03-31';
+
+    /**
+     * Indicates whether user engagement tracking should be disabled.
+     */
+    protected bool $disableUserEngagementTracking = false;
+
+    /**
+     * Create a new Azure Mailer Transport instance.
+     */
+    public function __construct(string $key, string $resourceName, bool $engagementTracking = true, $apiVersion = '2023-03-31', HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null, LoggerInterface $logger = null)
+    {
+        $this->resourceName = $resourceName;
+        $this->key = $key;
+        $this->apiVersion = empty($apiVersion) ? $this->apiVersion : $apiVersion;
+        $this->disableUserEngagementTracking = !$engagementTracking;
```

Comment:
```suggestion
        $this->engagementTracking = $engagementTracking;
```

No need to invert the value, engagementTracking is true or false

Reply:
The Azure Email Service documentation indicates that when this field is sent as true, it will disable the feature. The key name in the JSON is "userEngagementTrackingDisabled". See more at: https://learn.microsoft.com/en-us/rest/api/communication/dataplane/email/send?view=rest-communication-dataplane-2023-03-31&tabs=HTTP#request-body

Reply:
In this case lets change the DSN to `disable_tracking` and rename the constructor parameter to `$disableTracking`